src/mi/Lget_proc_info_in_range.c

```diff
@@ -0,0 +1,5 @@
+#define UNW_LOCAL_ONLY
```

Comment:
License?

Reply:
The local (L) source just include the global (G) ones with guard where I added the header. None of the other L files have the license: `cat src/mi/L*`.